drivers/SCsub

```diff
@@ -11,9 +11,7 @@ SConscript("windows/SCsub")
 # Sounds drivers
 SConscript("alsa/SCsub")
 SConscript("coreaudio/SCsub")
-SConscript("pulseaudio/SCsub")
-if env["platform"] == "windows":
-    SConscript("wasapi/SCsub")
+SConscript("wasapi/SCsub")
```

Comment:
`SConscript("pulseaudio/SCsub")` got accidentally removed.

Reply:
Fixed in upcoming push.